api/src/main/java/gov/cms/ab2d/api/security/SecurityConfig.java

```diff
@@ -71,15 +56,4 @@ public void configure(AuthenticationManagerBuilder authenticationManagerBuilder)
         authenticationManagerBuilder
                 .userDetailsService(customUserDetailsService);
     }
-
-    @Bean
-    public AccessTokenVerifier accessTokenVerifier() {
```

Comment:
Why was this method removed?

Reply:
It was moved to another class. The dependency was a circular dependency that was somehow working previously.